app/reports/mi-report/get-value.js

```diff
@@ -0,0 +1,18 @@
+const { PAYMENT_ENRICHED, PAYMENT_EXTRACTED } = require('../../constants/events')
+const { convertToPence } = require('../../currency-convert')
+
+const getValue = (events) => {
+  const enrichmentEvent = events.find(event => event.type === PAYMENT_ENRICHED)
+  if (enrichmentEvent) {
+    return enrichmentEvent.data.value
+  }
+  const extractedEvent = events.find(event => event.type === PAYMENT_EXTRACTED)
+  if (extractedEvent) {
+    return convertToPence(extractedEvent.data.value)
+  }
+  return events[0].data.value
```

Comment:
No need to convertToPence(events[0].data.value) here anymore?

Reply:
No, the only event that we are raising that isn't in pence is the extracted event.  But if we hit this line then we must not have one, so therefore can assume whatever event we do have is already in pence.